Buildings/Controls/OBC/Utilities/PIDWithAutotuning/Relay/ResponseProcess.mo

```diff
@@ -37,19 +37,20 @@ block ResponseProcess
     "A real signal of the normalized time delay"
     annotation (Placement(transformation(extent={{100,-10},{140,30}}),
         iconTransformation(extent={{100,-20},{140,20}})));
-  Buildings.Controls.OBC.Utilities.PIDWithAutotuning.Relay.OnOffPeriod onOffPer
-    "Calculate the length of the on period and the off period"
+protected
+  BaseClasses.OnOffPeriod                                              onOffPer
+    "Block that calculates the length of the on period and the off period"
     annotation (Placement(transformation(extent={{-80,0},{-60,20}})));
-  Buildings.Controls.OBC.Utilities.PIDWithAutotuning.Relay.HalfPeriodRatio halPerRatio
-    "Calculate the half period ratio"
+  BaseClasses.HalfPeriodRatio                                              halPerRatio
```

Comment:
full path, one line.

Reply:
The full path is used and the format is fixed